cypress/integration/links_test.js

```diff
@@ -60,36 +63,111 @@ describe("ensures introduction links and text are valid", () => {
 
   it("contains the error codes link", () => {
     // makes sure the error codes link goes to the right page
-    cy.get('*[class^="sc-iBzEeX sc-cOifOu dFWqin rUYTT redoc-markdown"]')
-      .contains("error code")
+    cy.contains("error code")
       .should("have.attr", "href")
       .and("include", "#tag/Errors");
   });
 
   it("contains the dashboard link", () => {
     // makes sure the dashboard link goes to the right page
-    cy.get('*[class^="sc-iBzEeX sc-cOifOu dFWqin rUYTT redoc-markdown"]')
-      .contains("Dashboard Settings")
+    cy.contains("Dashboard Settings")
       .should("have.attr", "href")
       .and("include", "dashboard.lob.com");
   });
 
   it("contains the dashboard settings link again", () => {
     // makes sure the other dashboard link goes to the right page
-    cy.get('*[class^="sc-iBzEeX sc-cOifOu dFWqin rUYTT redoc-markdown"]')
-      .contains("Settings")
+    cy.contains("Settings")
       .should("have.attr", "href")
       .and("include", "dashboard.lob.com");
   });
 
+  // no choice but to use classes for this
   it("contains the test & live environments link", () => {
     // makes sure the test & live environments link goes to the right page
+    // using a class is necessary for this because the element
+    // chain goes too deep otherwise
     cy.get('*[class^="sc-iBzEeX sc-cOifOu dFWqin rUYTT redoc-markdown"]')
       .contains("Test and Live Environments")
       .should("have.attr", "href")
       .and("include", "#tag/Test-and-Live-Environments");
   });
 
+  it("contains the Lob.com link", () => {
+    // makes sure the dashboard registry link goes to the right page
+    cy.contains("Lob.com")
+      .should("have.attr", "href")
+      .and("include", "https://dashboard.lob.com/#/register");
+  });
+
+  it("contains the keys link", () => {
+    // makes sure the keys link goes to the right page
+    cy.contains("settings")
+      .should("have.attr", "href")
+      .and("include", "https://dashboard.lob.com/#/settings/keys");
+  });
+
+  it("contains the SDK link", () => {
+    // makes sure the SDK link goes to the right page
+    cy.contains("Lob SDK")
+      .should("have.attr", "href")
+      .and("include", "#tag/SDKs-and-Tools");
+  });
+
+  // sc-hKFxyN dmghQN
```

Comment:
Comment for the future:

I don't love these . It would be nice if this code was abstracted out in a more readable way. Just off the top of my head, maybe create an object that maps a meaningful name to the value. So at the top of the file, or somewhere that makes sense if these tags are used across multiple files, declare an object like:

```
const PageSection_Enum = {
  Postcards: 'sc-hKFxyN dmghQN'
};
```

then use it like:
```
cy.get(`*[class^="${PageSection_Enum.Postcards}"]`)
```

Obviously adhere to our naming standards. My example code is consistent with a previous project that I was on